pallets/automation-time/src/weights.rs

```diff
@@ -62,6 +64,24 @@ impl<T: frame_system::Config> WeightInfo for AutomationWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(3 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
+	// Storage: Timestamp Now (r:1 w:0)
+	// Storage: AutomationTime Tasks (r:1 w:1)
+	// Storage: AutomationTime ScheduledTasks (r:1 w:1)
+	// TODO: this is placeholder, jzhou will fix
```

Comment:
just run the command at the top of this file but change the name of the file it outputs to. Then grab the results you want and throw them in here. The command outputs essentially this file minus how i expose the results

Reply:
TODO: need benchmarking code update

Reply:
also FYI, transfer is quite a bit heavier, as seen here. i copied over a task that was doing 90_000 and i couldn't accomplish without 120_000. https://github.com/OAK-Foundation/OAK-blockchain/pull/74/files#diff-8aa2921b8ffd2c8a5901003639e03bb443a2032d62c687606ea6a9f7bbcaf99cR554